packages/frontend/src/pages/scaling/liveness/props/getScalingLivenessView.tsx

```diff
@@ -130,11 +130,11 @@ function typeToDisplayType(
   anomaly: NonNullable<LivenessApiProject['anomalies']>[0],
 ) {
   switch (anomaly.type) {
-    case 'DA':
+    case 'batchSubmissions':
```

Comment:
I like this change, acronyms are nice in theory, later it is hard to use them while discussing code